pkg/api/install/install.go

```diff
@@ -19,99 +19,33 @@ limitations under the License.
 package install
 
 import (
-	"fmt"
-	"strings"
-
 	"github.com/golang/glog"
 
 	"k8s.io/kubernetes/pkg/api/latest"
-	"k8s.io/kubernetes/pkg/util/sets"
 
 	"k8s.io/kubernetes/pkg/api"
-	"k8s.io/kubernetes/pkg/api/meta"
-	"k8s.io/kubernetes/pkg/api/registered"
-	"k8s.io/kubernetes/pkg/api/unversioned"
-	"k8s.io/kubernetes/pkg/api/v1"
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-// userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
-
-const importPrefix = "k8s.io/kubernetes/pkg/api"
-
-var accessor = meta.NewAccessor()
-
 func init() {
-	groupMeta, err := latest.RegisterGroup("")
-	if err != nil {
-		glog.V(4).Infof("%v", err)
+	// this means that the entire group is disabled
+	if len(latest.ExternalVersions) == 0 {
```

Comment:
> !latest.IsEnabled()? (same for other places)

An init calling back into a public method?  Especially if its doesn't get reset, that makes a refactor dangerous for people not paying attention.